x-pack/plugins/security_solution/public/timelines/containers/active_timeline_context.ts

```diff
@@ -35,20 +39,42 @@ class ActiveTimelineEvents {
     this._activePage = activePage;
   }
 
-  getExpandedEvent() {
-    return this._expandedEvent;
+  getExpandedDetail() {
+    return this._expandedDetail;
   }
 
-  toggleExpandedEvent(expandedEvent: TimelineExpandedEventType) {
-    if (expandedEvent.eventId === this._expandedEvent.eventId) {
-      this._expandedEvent = {};
+  toggleExpandedDetail(expandedDetail: TimelineExpandedDetailType) {
+    const queryTab = TimelineTabs.query;
```

Comment:
Let's talk about this one, I do not think we need to do all of it. I will test it to make sure when I run the PR. I am commenting to not forget about it

Reply:
my bad, you are correct on this, this is need it.